app/screens/ScheduleScreen/ScheduleCard.tsx

```diff
@@ -29,14 +31,40 @@ const Header = ({ formattedEndTime, formattedStartTime, title, isPast }: HeaderP
   </View>
 )
 
-const Footer = ({ heading, subheading, isPast }: FooterProps) => (
-  <View style={$footerContainer}>
-    <Text preset="cardFooterHeading" style={isPast ? $pastFooterHeading : $footerHeading}>
-      {heading}
-    </Text>
-    <Text style={isPast ? $pastFooterSubheading : $footerSubheading}>{subheading}</Text>
-  </View>
-)
+const Footer = ({ heading, subheading, isPast, talkUrl, variant }: FooterProps) => {
+  return (
+    <View style={$footerContainer}>
+      <Text preset="cardFooterHeading" style={isPast ? $pastFooterHeading : $footerHeading}>
+        {heading}
+      </Text>
+      {isPast ? (
```

Comment:
So, I'd prefer if this footer didn't appear until after the conference ends. This `isPast` becomes true as soon as the talk is in the past, even if the conference is still happening. Can you rework this?